src/Beberlei/Metrics/Factory.php

```diff
@@ -31,49 +31,49 @@
     /**
      * Create Metrics Collector Instance
      *
-     * @param string $type
-     * @param array $options
+     * @param  string                                $type
+     * @param  array                                 $options
```

Comment:
I would rather separate the params and returns rather than aligning the names that far